### DISCUSSION TITLE
control a light via radio

### DISCUSSION THREAD

**Discussion by siiimon1** [rtl-airband/RTLSDR-Airband#324] [Q&A]
Hi,

I´d like to control a light on our airfield via radio (from the aircraft).
So is it possible to count the number of cliks on the PTT button in a certain time?
As soon as the number is reached (e.g. 5 clicks in 10s) a GPIO port should be opend. 

Thanks!

**Comment by LoganSound** [comment]
This intrigues me! I am using "inotifywait" to alert when new files are created but I think more simply you could use the log, "inotifywait" or another tool could trigger when the log changes, a script could count sequential log items within a fixed interval and trigger your GPIO?

**Reply by siiimon1** [reply]
good idea, I will try

**Comment by ScanCaster** [comment]
> I´d like to control a light on our airfield via radio (from the aircraft). So is it possible to count the number of cliks on the PTT button in a >  >certain time? As soon as the number is reached (e.g. 5 clicks in 10s) a GPIO port should be opend.

I would review the following if this is for an airfield. 

"Remote controls may be designed using a fixed-wire method, or radio control with L-854 equipment per https://www.faa.gov/regulations_policies/advisory_circulars/index.cfm/go/document.information/documentID/1031147  "

Just FYI heads up...

**Reply by siiimon1** [reply]
Thanks, but the airfield is in Germany, so FAA regulations won´t apply. 
And the light is not safety-relevant, so it does not matter ;)

**Reply by remsiux** [reply]
Did you manage to get this working for you?
